README.md

```diff
@@ -173,6 +173,7 @@ Currently supported distributions are:
 - [asciinema-edit](https://github.com/cirocosta/asciinema-edit)
 - [awless](https://github.com/wallix/awless)
 - [aws-iam-authenticator](https://github.com/kubernetes-sigs/aws-iam-authenticator)
+- [aws-vault](https://github.com/kubernetes-sigs/aws-iam-authenticator)
```

Comment:
I guess the proper URL is https://github.com/99designs/aws-vault right ?

Reply:
@leucos Sorry, fixed

Reply:
No problem ! thanks :+1: